pandas/tests/frame/methods/test_fillna.py

```diff
@@ -708,6 +708,34 @@ def test_single_block_df_with_horizontal_axis(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    def test_fillna_with_multi_index_frame(self):
+        # GH 47649
+        pdf = DataFrame(
+            {
+                ("x", "a"): [np.nan, 2.0, 3.0, 4.0, np.nan, 6.0],
```

Comment:
Could you simplify the DataFrame? 2 rows should be sufficient.

Also please add a test with ea dtypes

Reply:
Sure, could you be more specific about what dtypes we want in the test?

Reply:
Int64 for example